app/views/eligibility-tool/employment-status.html

```diff
@@ -26,7 +26,7 @@
   {% endset %}
 
   {% set helpText %}
-    Read about <a href="https://www.gov.uk/employment-status">employment statuses</a> if you aren’t sure which one applies.
+    Read about <a href="https://www.gov.uk/employment-status">employment statuses</a> if you're not sure which one applies.
```

Comment:
I don't think it matters in the prototype here, but if you are making changes in the beta later, apostrophes and quotes are different characters. You can type the apostrophe character in VSCode using alt + ]